src/app/sidebar/shared/styles.ts

```diff
@@ -0,0 +1,6 @@
+export const Colors = {
+    black: "black",
+    grey: "grey",
+    orange200: "#FBD38D",
+    white: "white"
+}
```

Comment:
I get the idea and agree we shouldn't be using random hex and color constants however those values are actually fully typed and pulled from a global theme. Maybe we should define a set of colours but since we haven't using the built-in ones seems fine for now. 

This has documentation of how the global theme (where something like what you defined), might make more sense. 
https://chakra-ui.com/docs/features/global-styles